zio-http/jvm/src/test/scala/zio/http/endpoint/openapi/OpenAPIGenSpec.scala

```diff
@@ -231,7 +231,7 @@ object OpenAPIGenSpec extends ZIOSpecDefault {
       .outError[NotFoundError](Status.NotFound)
 
   private val endpointWithAuthScopes =
```

Comment:
The public API before was okay. Just the internal impl. I wanted you to change. Sorry, I was maybe not clear enough